R/protein_quant.R

```diff
@@ -419,7 +419,7 @@ protein_quant <- function (pepData, method, isoformRes = NULL,
       dplyr::relocate(dplyr::any_of(emeta_cols), .after = n_peps_used) %>%
       # Only keep distinct combinations of the columns that are kept.
       dplyr::distinct(dplyr::all_of(.)) %>%
-      data.frame()
+      data.frame(check.names = FALSE)
```

Comment:
should test if this works with a check.names = TRUE object.  I think these data.frame calls should take the value of `attributes(object)$check_names`.  There's a local variable called 'fijate' (no idea) in the function that is holding this value.

Reply:
Ok, I actually tested this and am now confused... Setting check.names = TRUE in the object construction doesn't actually change any of the data frames in the object, and fails when we get to protein_quant() due to this line:  [609](https://github.com/pmartR/pmartR/blob/50f1e02f5d1ded5b99f3aea2e0034fae57bf8d89/R/protein_quant.R#L609)  

It seems the default of check.names=TRUE is **never** appropriate.